src/main/java/uk/gov/ch/service/update/impl/CorporateTrusteesServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.ch.service.update.impl;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.ch.OracleQueryApplication;
+import uk.gov.ch.exception.TrustDataCountNotFoundException;
+import uk.gov.ch.model.update.CorporateTrusteeData;
+import uk.gov.ch.repository.update.CorporateTrusteesRepository;
+import uk.gov.ch.service.update.CorporateTrusteesService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.logging.util.DataMap;
+
+
+@Service
+public class CorporateTrusteesServiceImpl implements CorporateTrusteesService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            OracleQueryApplication.APPLICATION_NAME_SPACE);
+
+    @Autowired
+    private CorporateTrusteesRepository corporateTrusteesRepository;
+
+    @Override
+    public List<CorporateTrusteeData> getCorporateTrusteeData(String trustId)
+            throws TrustDataCountNotFoundException {
+        List<CorporateTrusteeData> corporateTrusteeDataList = corporateTrusteesRepository.getCorporateTrustees(
+                trustId);
+
+        DataMap dataMap = new DataMap.Builder().build();
```

Comment:
I would maybe move this logging into the else block below, just in case corporateTrusteeDataList is null when you call corporateTrusteeDataList.size()

Reply:
If you believe that's possible then a `NullPointerException` will be raised on the if statement instead 

Either way, it will have the same result.

It may be best to just put a null check above if you think a null is likely